### PR TITLE
Feature/store games

### PR DESCRIPTION
**Pull request by AbrahamAlcaina**

### CODE REVIEW

File: client/src/js/states/main.js [hunk 1]
```diff
@@ -1,65 +1,98 @@
...
+    // connect with FB (user, locale...)
   }
 
   create() {
     this.initialize();
   }
 
   initialize() {
+    //solve blurry pixels
+    this.game.renderer.renderSession.roundPixels = true;
```
Comment (bernatmv) [inline client/src/js/states/main.js:22]: I moved this to the boot state

File: client/src/js/utils.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export const getURLParameter = name => decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search) || [, ''])[1].replace(/\+/g, '%20')) || null;
```
Comment (bernatmv) [inline client/src/js/utils.js:1]: 1 line O_o ?

File: server/commands/joinGame.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const joinGame = (io, socket, games) => socket.on('joinGame', function(data) {
+    const game = games[data.gameId];
+    console.log('join', game, data);
+    if (!game || game.guest) {
```
Comment (bernatmv) [inline server/commands/joinGame.js:4]: what does the `game.guest` condition looks for?

File: server/commands/startGame.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const uuid = require('node-uuid');
+
+const startGame = (io, socket, games) => socket.on('startGame', function(data) {
+    if (!data) {
+        io.to(socket.id).emit('error', {
+            message: 'no user data on game starts'
```
Comment (bernatmv) [inline server/commands/startGame.js:6]: `game start`, remove the last `s`

File: server/commands/startGame.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        });
+    }
+    const gameId = uuid.v1();
+    const game = {
+        numberOfRounds: data.rounds || 3,
```
Comment (bernatmv) [inline server/commands/startGame.js:11]: porperties file for the `3`?

**Review by bernatmv — CHANGES_REQUESTED** [review]

File: server/commands/joinGame.js (hunk 3 above)
Comment (AbrahamAlcaina) [reply server/commands/joinGame.js:4]: That only exists 1 guest.
The second time that enters in the same url we return an error

File: client/src/js/utils.js (hunk 2 above)
Comment (AbrahamAlcaina) [reply client/src/js/utils.js:1]: I don't have idea that this regex do. 
I can splite it but it don't understand it 😂

**Review by bernatmv — APPROVED** [review]

**Pull request merged**